discovery_engine_core/web-api/src/bin/ingestion.rs

```diff
@@ -310,78 +335,47 @@ fn serialize_to_ndjson(
     Ok(bytes.freeze())
 }
 
-#[derive(Debug)]
-struct TooManyDocumentsError;
-impl Reject for TooManyDocumentsError {}
+#[derive(Error, Debug, DisplayDoc)]
+enum Error {
+    /// Too many documents send to ingestion system
+    TooManyDocuments,
```

Comment:
maybe we could add a count param here and a better error msg, like here https://github.com/xaynetwork/xayn_discovery_engine/blob/main/discovery_engine_core/web-api/src/models.rs#L53-L54